static/src/javascripts-legacy/projects/common/modules/experiments/tests/acquisitions-epic-election-interactive.js

```diff
@@ -0,0 +1,56 @@
+define([
+    'common/modules/commercial/contributions-utilities',
+    'lib/geolocation',
+    'lodash/utilities/template',
+    'lib/config',
+    'raw-loader!common/views/acquisitions-epic-control.html'
+], function (
+    contributionsUtilities,
+    geolocation,
+    template,
+    config,
+    epicControlTemplate
+) {
+    return contributionsUtilities.makeABTest({
+        id: 'AcquisitionsEpicElectionInteractive',
+        campaignId: 'epic_election_interactive_end',
+
+        start: '2017-05-22',
+        expiry: '2017-07-03',
+
+        author: 'Sam Desborough',
+        description: 'This places the epic underneath UK election-related interactives',
+        successMeasure: 'Member acquisition and contributions',
+        idealOutcome: 'Our wonderful readers will support The Guardian in this time of need!',
+
+        audienceCriteria: 'All',
+        audience: 1,
+        audienceOffset: 0,
+
+        showForSensitive: true,
+
+        pageCheck: function(page) {
+            return page.keywordIds.includes('general-election-2017') && page.contentType === 'Interactive';
+        },
+
+        variants: [
+            {
+                id: 'control',
+                isUnlimited: true,
+
+                insertAtSelector: '.content-footer',
```

Comment:
is this guaranteed to be at the bottom of every interactive? I assume it is but just checking